src/pr-trigger.ts

```diff
@@ -5,158 +5,146 @@ import { process as computeActions } from "./compute-pr-actions";
 import { executePrActions } from "./execute-pr-actions";
 import { mergeCodeOwnersOnGreen } from "./side-effects/merge-codeowner-prs";
 import { HttpRequest, Context } from "@azure/functions";
-import { Webhooks, EventPayloads } from "@octokit/webhooks";
+import { createEventHandler, EmitterWebhookEvent, verify } from "@octokit/webhooks";
 
 const prHandlers: { [prNumber: number]: () => void } = {};
 
 export async function httpTrigger(context: Context, req: HttpRequest) {
-
-    if (!(process.env["BOT_AUTH_TOKEN"] || process.env["AUTH_TOKEN"])) {
-        throw new Error("Set either BOT_AUTH_TOKEN or AUTH_TOKEN to a valid auth token");
-    }
-
```

Comment:
Always false, thrown in `graphql-client.ts` https://github.com/DefinitelyTyped/dt-mergebot/blob/f4a802020f366e4b85dfeb20eba68bb6fe33a6a0/src/graphql-client.ts#L35